modules/express/test/unit/clientRoutes/consolidateAccount.ts

```diff
@@ -89,11 +68,11 @@ describe('Consolidate account', () => {
       .should.be.rejectedWith('consolidate address must be an array of addresses');
   });
 
-  function createConsolidateMocks(res, coin = CoinFamily.ALGO) {
+  function createConsolidateMocks(res, allowsAccountConsolidations = true) {
```

Comment:
```suggestion
  function createConsolidateMocks(res, allowsAccountConsolidations = false) {
```

Reply:
Since the default is false and true is an explicit action

Reply:
fixed